### PR TITLE
FFM-6615: Adds missed relnotes for android sdk 1.0.19

### PR DESCRIPTION
**Pull request by StrawberryPad**

### CODE REVIEW

File: release-notes/feature-flags.md [hunk 1]
```diff
@@ -214,8 +214,13 @@ This release does not include early access features.
...
-  - You can now provide the SDK with a set of evaluations that it can serve instantly upon initialization. For more information about this, go to [the SDK's readme file.](https://github.com/harness/ff-javascript-client-sdk/blob/main/README.md) (FFM-6489) 
+- The Javascript client SDK has been updated to version 1.8.0 and includes the following change:
+  - You can now provide the SDK with a set of evaluations that it can serve instantly upon initialization. For more information about this, go to [the SDK's readme file.](https://github.com/harness/ff-javascript-client-sdk/blob/main/README.md) (FFM-6489)
+
+- The Android client SDK has been updated to version 1.0.19 and includes the following changes:
+  - A new event was added, `SSE_RESUME`, which fires if the application loses and regains internet. When the event fires: 
+    - The SDK internally reloads all applications into cache.
+    - Applications can listen to this event to ensure event listeners don't miss any streamed events during periods of downtime. For example, an application can call a variation function to get the latest evaluation result.
```
Comment (erdirowlands) [inline release-notes/feature-flags.md:223]:
```suggestion
    - Applications can listen to this event to ensure event listeners don't miss any streamed events during periods of downtime.
```

Comment (erdirowlands) [reply release-notes/feature-flags.md:223]: Suggestion: remove the "for example" sentence which I included in the public release note summary. I don't think it's needed

File: release-notes/feature-flags.md [hunk 2]
```diff
@@ -214,8 +214,13 @@ This release does not include early access features.
...
 
-- The Javascript SDK has been updated to version 1.8.0 and includes the following change:
-  - You can now provide the SDK with a set of evaluations that it can serve instantly upon initialization. For more information about this, go to [the SDK's readme file.](https://github.com/harness/ff-javascript-client-sdk/blob/main/README.md) (FFM-6489) 
+- The Javascript client SDK has been updated to version 1.8.0 and includes the following change:
+  - You can now provide the SDK with a set of evaluations that it can serve instantly upon initialization. For more information about this, go to [the SDK's readme file.](https://github.com/harness/ff-javascript-client-sdk/blob/main/README.md) (FFM-6489)
+
+- The Android client SDK has been updated to version 1.0.19 and includes the following changes:
+  - A new event was added, `SSE_RESUME`, which fires if the application loses and regains internet. When the event fires: 
+    - The SDK internally reloads all applications into cache.
```
Comment (erdirowlands) [inline release-notes/feature-flags.md:222]:
```suggestion
    - The SDK internally reloads all feature config into cache.
```

Comment (erdirowlands) [reply release-notes/feature-flags.md]: A bad typo from me in my initial release notes. Corrected above.

**Review by erdirowlands — APPROVED** [review]
Looks good! Great to have these missing relnotes caught up.

File: release-notes/feature-flags.md [hunk 3]
```diff
@@ -214,8 +214,13 @@ This release does not include early access features.
...
   - You can now provide the SDK with a set of evaluations that it can serve instantly upon initialization. For more information about this, go to [the SDK's readme file.](https://github.com/harness/ff-react-client-sdk/blob/main/README.md) (FFM-6578) 
 
-- The Javascript SDK has been updated to version 1.8.0 and includes the following change:
-  - You can now provide the SDK with a set of evaluations that it can serve instantly upon initialization. For more information about this, go to [the SDK's readme file.](https://github.com/harness/ff-javascript-client-sdk/blob/main/README.md) (FFM-6489) 
+- The Javascript client SDK has been updated to version 1.8.0 and includes the following change:
+  - You can now provide the SDK with a set of evaluations that it can serve instantly upon initialization. For more information about this, go to [the SDK's readme file.](https://github.com/harness/ff-javascript-client-sdk/blob/main/README.md) (FFM-6489)
+
+- The Android client SDK has been updated to version 1.0.19 and includes the following changes:
+  - A new event was added, `SSE_RESUME`, which fires if the application loses and regains internet.  When the event fires:
```
Comment (SmritiSatya) [inline release-notes/feature-flags.md:221]: If this statement is rephrased, will the meaning change? Here is the rephrased statement:
A new event was added, `SSE_RESUME`, which is fired when the application loses and regains internet.  When the event is fired:

**Review by SmritiSatya — APPROVED** [review]

File: release-notes/feature-flags.md (hunk 3 above)
Comment (StrawberryPad) [reply release-notes/feature-flags.md:221]: Thanks, Smriti. Even though this isn't perfectly grammatically correct, it does match speech patterns (which is a tenet of content usability... like having a dangling preposition). For example, you could say "choose the item you want to add a tag to" - it's technically incorrect but used in speech (more commonly than "to which you want to add a tag"). I let these go (sparingly!) if it sounds like what a tech user might easily say.  <long-winded reply>

**Pull request merged**